src/Orchard.Cms.Web/Modules/Orchard.Setup/Annotations/SiteNameValidAttribute.cs

```diff
@@ -23,9 +18,9 @@ public override bool IsValid(object value)
         public override string FormatErrorMessage(string name)
         {
             if (string.IsNullOrWhiteSpace(_value))
-                return T("Site name is required.");
+                return "Site name is required.";
 
-            return T("Site name can be no longer than {0} characters.", Maximum);
+            return string.Format("Site name can be no longer than {0} characters.", Maximum);
```

Comment:
Don't use `String.Format`, prefer string interpolation `$"Site name can be no longer than {Maximum} characters."`